examples/Huggingface_Transformers/requirements.txt

```diff
@@ -0,0 +1,4 @@
+torch
+transformers
```

Comment:
Apart from `transformers` all other packages are already part of TorchServe requirement files.

Reply:
@harshbafna Should we remove all other packages from the list or let it be there ?

Reply:
@shrinath-suresh: Yes, that would be ideal. Specially torch package as it will be completely dependent on the environment (CPU/GPU) and CUDA version in case of GPU.

Reply:
Done. I have removed all the packages except `transformers`